static/app/views/organizationGroupDetails/groupEventDetails/groupEventDetails.tsx

```diff
@@ -228,6 +228,8 @@ class GroupEventDetails extends Component<GroupEventDetailsProps, State> {
     const {activity: activities} = group;
     const mostRecentActivity = getGroupMostRecentActivity(activities);
 
+    const hasReplay = Boolean(event?.tags?.find(({key}) => key === 'replayId')?.value);
```

Comment:
this should be guarded by the ui feature flag. 
right now, if the flag is disabled then we'll get the button but no section on the page.